### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by Zehuan-Wang**
add a function that could record and review past queries,
planning to continue with Delete and Edit, and improve the UI

### CODE REVIEW

**Comment by Zehuan-Wang** [comment]
I fixed some bugs and there is still one main issue remain, which is when the window opens, the window of Wox keep showing and cannot hide.

This is my first time to try to make a contribution and pull a request! Sorry for the fact that I am inexperienced. I'm happy to learn new things!

File: history.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import json
+import sqlite3
+
+
+class QueryDB:
+    def __init__(self):
+        self.conn = ''
```
Comment (Mnikley) [inline history.py]: initialize with None instead

File: app.py
Comment (Mnikley) [inline app.py:1]: i noticed a breaking bug when no API key is given and/or the API keys quota is exceeded - will check in the next couple of days for the exact line and come back to you

**Comment by Mnikley** [comment]
Hi Marco-Ray, thanks for your contribution! I like the idea in general and think that both the SQLite history and the related frontend is well written - the problem where the Wox input bar stays open until the workload is finished is known to me, but i did not find a solution to circumvent this, yet. The subprocess.Popen call with the CREATE_NO_WINDOW flag is a nice addition, however. I will try to continue with the check in the next couple of days. Thank you!

**Comment by Mnikley** [comment]
i stopped the intervalRunner after retrieving an item from the sqlite database (line 193, index.html) - currently i am unable to restart the intervalRunner or to refresh the page after a new API call is made.

**Review by Mnikley — APPROVED** [review]
adapted code slightly; known bug: completed queries are added to the history side-panel twice due to async nature of application

**Pull request merged**